### PR TITLE
refactor: rename ValueState values

### PR DESCRIPTION
**Pull request by ilhan007**
The base ValueState enum values have been changed from `ValueState.Warning`, `ValueState.Error` and `ValueState.Success` to `ValueState.Critical`, ValueState.Negative and `ValueState.Positive`, causing changes in multiple components supporting valueState property (+ properties of similar type as `Dialog.state`, `StandardListItem.highlight` and `MessageStrip.design`).

BREAKING CHANGE: If you previously used `ValueState.Warning`, `ValueState.Error` or `ValueState.Success`, start using `ValueState.Critical`, `ValueState.Negative` and `ValueState.Positive` respectively. All components with `valueState` property are also affected:
| < 2.0 | >= 2.0|
--------------------|-------------|
|**ui5-checkbox** value-state=**"Error/Warning/Success"** | **ui5-checkbox** value-state=**"Negative/Critical/Positive"**|
|**ui5-combobox** value-state=**"Error/Warning/Success"** |**ui5-combobox** value-state=**"Negative/Critical/Positive"**
|**ui5-datepicker** value-state=**"Error/Warning/Success"** |**DatePicker** value-state=**"Negative/Critical/Positive"**
|**ui5-date-time-picker** value-state=**"Error/Warning/Success"** |**ui5-date-time-picker** value-state=**"Negative/Critical/Positive"**
|**ui5-date-range-picker** value-state=**"Error/Warning/Success"** |**ui5-date-range-picker** value-state=**"Negative/Critical/Positive"**
|**ui5-dialog** state="Error/Warning/Success" |**ui5-dialog** state==**"Negative/Critical/Positive"**|
|**ui5-file-uploader** value-state=**"Error/Warning/Success"** |**ui5-file-uploader** value-state=**"Negative/Critical/Positive"**
|**ui5-input** value-state=**"Error/Warning/Success"** |**ui5-input** value-state=**"Negative/Critical/Positive"**
|**ui5-li** highlight=**"Error/Warning/Success"** |**ui5-li** highlight=**"Negative/Critical/Positive"**|
|**ui5-li** additional-text-state=**"Error/Warning/Success"** | **ui5-li** additional-text-state=**"Negative/Critical/Positive"**|
|**ui5-message-strip** design=**"Warning"** | **ui5-message-strip** design=**"Critical"** |
|**ui5-multi-input** value-state=**"Error/Warning/Success"** |**ui5-multi-input** value-state=**"Negative/Critical/Positive"**
|**ui5-multi-combobox** value-state=**"Error/Warning/Success"** |**ui5-multi-combobox** value-state=**"Negative/Critical/Positive"**
|**ui5-progress-indicator** value-state=**"Error/Warning/Success"** |**ui5-progress-indicator** value-state=**"Negative/Critical/Positive"**
|**ui5-radio-button** value-state=**"Error/Warning/Success"** |**ui5-radio-button** value-state=**"Negative/Critical/Positive"**
|**ui5-select** value-state=**"Error/Warning/Success"** |**ui5-select** value-state=**"Negative/Critical/Positive"**|
|**ui5-step-input** value-state=**"Error/Warning/Success"** |**ui5-step-input** value-state=**"Negative/Critical/Positive"**|
|**ui5-textarea** value-state=**"Error/Warning/Success"** |**ui5-textarea** value-state=**"Negative/Critical/Positive"**
|**ui5-time-picker** value-state=**"Error/Warning/Success"** |**ui5-time-picker** value-state=**"Negative/Critical/Positive"**|
|**ui5-tree-item** highlight=**"Error/Warning/Success"** |**ui5-tree-item**highlight=**"Negative/Critical/Positive"**|
|**ui5-tree-item** additional-text-state=**"Error/Warning/Success"** |**ui5-tree-item** additional-text-state=**"Negative/Critical/Positive"|**

Related to: https://github.com/SAP/ui5-webcomponents/issues/8461

### CODE REVIEW

File: packages/main/src/types/HighlightTypes.ts [hunk 1]
```diff
@@ -12,17 +12,17 @@ enum HighlightTypes {
 	/**
 	 * @public
 	 */
-	Success = "Success",
+	Success = "Positive",
```
Comment (nnaydenow) [inline packages/main/src/types/HighlightTypes.ts:15]: Change enum keys to the new ones.

File: packages/main/src/TreeItemBase.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ class TreeItemBase extends ListItem {
 	/**
 	 * Defines the state of the `additionalText`.
 	 *
-	 * Available options are: `"None"` (by default), `"Success"`, `"Warning"`, `"Information"` and `"Error"`.
+	 * Available options are: `"None"` (by default), `"Positive"`, `"Critical"`, `"Information"` and `"Critical"`.
```
Comment (nnaydenow) [inline packages/main/src/TreeItemBase.ts:154]: We have `Critical` twice. Maybe you though `Negative` on the second one.

**Review by nnaydenow — CHANGES_REQUESTED** [review]
To things to discuss:
- some of the css classes related to value states are renamed and others not
- should we find and replace all value state translations texts for consistency?

**Comment by ilhan007** [comment]
- the translation texts keys we can change, but the values should remain the same for now, they will be update later in the specs
- the classes we can change, I thought it will become huge change and focused on the API changes

**Review by nnaydenow — APPROVED** [review]

**Pull request merged**